IdentityServer4.Contrib.RedisStore/IdentityServer4.Contrib.RedisStore.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
that will prevent people who are still on IdServer4 3.0.* from adopting the latest feature

Reply:
Yes of course